### DISCUSSION TITLE
Angular 17 support ?

### DISCUSSION THREAD

**Discussion by euskadi31** [vmware-clarity/ng-clarity#1048] [General]
Hello, do you plan to support Angular 17 for @clr/angular?

Thank you

**Comment by LynxDelli** [comment]
Our plan is ng-clarity 17 to support Angular 17, the expected release date is Feb-Mach 2024.

**Reply by mcosta74** [reply]
I just wanna say I was able to use @clr/angular with Angular v17 installing with `npm install --force`

**Reply by euskadi31** [reply]
Hello, does this release support Angular 17?

https://github.com/vmware-clarity/ng-clarity/releases/tag/v17.0.0

Thank

**Reply by LynxDelli** [reply]
Yes, the last release v17.0.0 Supports Angular 17

**Comment by mcosta74** [comment]
I can confirm that. It works with Angular v17.2

Il Mar 20 Feb 2024, 13:27 Delchina Angelova ***@***.***> ha
scritto:
